### PR TITLE
Add non-normative language to Co-ordinated Provisioning section

### PR DESCRIPTION
**Pull request by jischr**
It seemed like the CP section needed to be rounded out a bit with examples of transmission methods.
This non-normative language felt like a good start.

### CODE REVIEW

File: draft-ietf-scim-events-06.xml [hunk 1]
```diff
@@ -1627,9 +1633,12 @@ Location:
...
                     When sharing information between parties, CP Events minimize the information shared in each message and
-                    require the Security Event Receiver to callback to the Event Publisher to retrieve more information.
-                    In this way, the Event Publisher is able to have regular access to information through normal
-                    SCIM protocol access restrictions.
+                    require the Security Event Receiver to receive more information from the Event Publisher as needed.
+                    In this way, the Event Receiver is able to have regular access to information through normal
+                    SCIM protocol access restrictions. The Event Receiver and Publisher may agree to communicate these
+                    updates through a variety of transmission methods such as push and pull based HTTP like in <xref target="RFC8935"/>,
+                    <xref target="RFC8936"/>, or HTTP GET (see <xref target="asyncRequest"/>), streaming technologies
+                    (e.g., Kaftka or Kinesis), or via webhooks like in the <xref target="SSF" format="default">Shared Signals Framework</xref>.
```
Comment (ahaeber) [inline draft-ietf-scim-events-06.xml]: Spelling error: "Kaftka" probably should be Kafka

File: draft-ietf-scim-events-06.xml [hunk 2]
```diff
@@ -1297,7 +1297,13 @@ Location:
                 <name>Informative References</name>
                 <xi:include href="https://www.rfc-editor.org/refs/bibxml/reference.RFC.8935.xml"/>
                 <xi:include href="https://www.rfc-editor.org/refs/bibxml/reference.RFC.8936.xml"/>
-
+                <reference anchor="SSF">
+                    <front>
+                        <title>Shared Signals Framework</title>
+                        <author><organization>OpenId Foundation</organization></author>
```
Comment (aaronpk) [inline draft-ietf-scim-events-06.xml]:
```suggestion
                        <author><organization>OpenID Foundation</organization></author>
```

File: draft-ietf-scim-events-06.xml (hunk 1 above)
Comment (aaronpk) [inline draft-ietf-scim-events-06.xml]:
```suggestion
                    (e.g., Kafka or Kinesis), or via webhooks like in the <xref target="SSF" format="default">Shared Signals Framework</xref>.
```

Comment (jischr) [inline draft-ietf-scim-events-06.xml:1641]:
```suggestion
                    (e.g., Kafka or Kinesis), or via webhooks like in the <xref target="SSF" format="default">Shared Signals Framework</xref>.
```

**Review by independentid — APPROVED** [review]

**Pull request merged**